src/Factory.php

```diff
@@ -302,6 +312,13 @@ function($value) {
             $value = $value->withoutPersisting();
         }
 
+        // Check if the attribute is cascade persist
+        $relationField = $this->relationshipField($value) ?? $this->inverseRelationshipField($value);
```

Comment:
This is causing the `RuntimeException: Foundry was booted without doctrine.` failure. It is possible to use foundry without doctrine being available. It just creates the object w/o persisting. These functions try to access doctrine which causes the exception.

Reply:
Thanks, I add a condition with `self::configuration()->hasManagerRegistry()`.